eras/shelley/impl/src/Cardano/Ledger/Shelley/PParams.hs

```diff
@@ -321,22 +335,30 @@ upgradeProposedPPUpdates ::
 upgradeProposedPPUpdates args (ProposedPPUpdates ppus) =
   ProposedPPUpdates $ upgradePParamsUpdate args <$> ppus
 
-ppMinFeeA :: EraPParams era => PParam era
-ppMinFeeA =
+ppMinFeeFactor :: EraPParams era => PParam era
+ppMinFeeFactor =
```

Comment:
To make it consistent across the board. Let's call it `ppTxFeePerByte` instead of `ppMinFeeFactor`. This will make it consistent with how cardano-api names it.
```suggestion
ppTxFeePerByte =
```
Naturally this comment applies to all the occurrences in this PR